cmd/zed/token.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"os"
+
+	"github.com/jzelinskie/cobrautil"
+	"github.com/jzelinskie/stringz"
+	"github.com/spf13/cobra"
+
+	"github.com/authzed/zed/internal/printers"
+	"github.com/authzed/zed/internal/storage"
+)
+
+var tokenCmd = &cobra.Command{
+	Use:   "token <subcommand>",
+	Short: "manage the API tokens stored in your keychain",
+}
+
+var tokenListCmd = &cobra.Command{
+	Use:               "list",
+	Short:             "list all the API Tokens from your keychain",
+	Args:              cobra.ExactArgs(0),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              tokenListCmdFunc,
+}
+
+var tokenSaveCmd = &cobra.Command{
+	Use:               "save <system> <token>",
+	Short:             "save an API Token to your keychain",
+	Args:              cobra.ExactArgs(2),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              tokenSaveCmdFunc,
+}
+
+var tokenDeleteCmd = &cobra.Command{
+	Use:               "delete <system>",
+	Short:             "delete an API Token from your keychain",
+	Args:              cobra.ExactArgs(1),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              tokenDeleteCmdFunc,
+}
+
+var tokenUseCmd = &cobra.Command{
+	Use:               "use <system>",
+	Short:             "use an API Token for future commands",
+	Args:              cobra.ExactArgs(1),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              tokenUseCmdFunc,
+}
+
+func tokenListCmdFunc(cmd *cobra.Command, args []string) error {
+	tokens, err := storage.DefaultTokenStore.List(!cobrautil.MustGetBool(cmd, "reveal-tokens"))
+	if err != nil {
+		return err
+	}
+
+	var rows [][]string
+	for _, token := range tokens {
+		rows = append(rows, []string{
+			token.Name,
```

Comment:
Maybe something more descriptive than `name` for the token, since it becomes the prefix given?